src/nspawn/nspawn-mount.c

```diff
@@ -271,7 +275,13 @@ int mount_sysfs(const char *dest) {
                 return log_error_errno(errno, "Failed to remove %s: %m", full);
 
         x = prefix_roota(top, "/fs/kdbus");
-        (void) mkdir(x, 0755);
+        (void) mkdir_p(x, 0755);
+
+	// Create mountpoint for cgroups. Otherwise we are not allowed since we remount /sys read-only.
+	if (cg_ns_supported()) {
+		x = prefix_roota(top, "/fs/cgroup");
+		(void) mkdir_p(x, 0755);
+	}
```

Comment:
This is needed to deal with the situation where a `tmpfs` is mounted by the `inner_child()`. Since we remount `/sys` read-only at the end of this function we need to create the mountpoint for `/sys/fs/cgroup` now..